garden/src/api_demos/SearchWidget.tsx

```diff
@@ -1,33 +0,0 @@
-import React, { useState } from "react";
-
-import { GARDEN_INDEX_URL, SEARCH_SCOPE } from "../constants"
-import { fetchWithScope } from "../globusHelpers"
-
-export default function GardenWidget() {
-    const [status, setStatus] = useState<string>("");
-
-    const callSearch = async () => {
-        try {
-            const response = await fetchWithScope(SEARCH_SCOPE, GARDEN_INDEX_URL + "/search?q=\"ionization\"");
-        
-            if (!response.ok) {
-                throw new Error('Network response was not ok');
-            }
-            const content = await response.text()
-            setStatus(content);
-        } catch (error) {
-            setStatus("failed");
-        }
-    };
-
-    const handleClick = () => {
-        callSearch();
-    };
-
-    return (
-        <div>
-        <button onClick={handleClick}>Find some gardens</button>
-        <p>{status}</p>
-        </div>
-    );
-}
```

Comment:
This was just me demoing to the interns how to use Search. Now that they've gone and implemented it this is cruft.